docs/coding-style.md

```diff
@@ -105,8 +105,9 @@ Variable name `sut` (System Under Test) is recommended in unit tests that really
             // Action
         }
     ```
-* Expression body of arrow property should be on the same line, as the property declaration. It should be on next line only when S103 is violated.
-* Expression body of method should be on the same line only for trivial cases: literal or simple identifier. Member access, indexer, invocation and other complex structures should be on the next line.
+* When using arrow property or method, the arrow is always on the same line with the declaration. Regarding the body:
```

Comment:
Nitpick:
```suggestion
* When using an arrow-bodied property or method, the arrow token should be always on the same line as the declaration. Regarding the body:
```

Reply:
I would call them `arrow property or method` as the original. The `-bodied` suffix is used in `expression-bodied` case and that wording isn't intuitively clear to me.

Reply:
`token must be` instead of `token should be`, see [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) for this kind of wording